README.md

```diff
@@ -1 +1,17 @@
-# projectportal-email-service
\ No newline at end of file
+# projectportal-email-service
+
+Project Portal email service will run on the production internal cluster, `bkpi-ranch`.
+
+A test version has also been set up on the qa internal cluster, `qbkpi-ranch`.
+
+To access either cluster use the following command 
+```kubectx bkpi-ranch``` or ```kubectx qbkpi-ranch```
```

Comment:
when I run `kubectx qbkpi-ranch` then `kubens` I get: `error: the server doesn't have a resource type "namespaces" error getting namespace list`